charts/trino/values.yaml

```
@@ -32,6 +32,10 @@ server:
       type: "UseG1GC"
       g1:
         heapRegionSize: "32M"
+  autoscaler:
```

nit: to align to convention in most other charts I've seen. 
```suggestion
  autoscaling:
```